src/main/groovy/com/lesfurets/jenkins/unit/declarative/StageDeclaration.groovy

```diff
@@ -1,31 +1,53 @@
 package com.lesfurets.jenkins.unit.declarative
 
 
+import static com.lesfurets.jenkins.unit.declarative.GenericPipelineDeclaration.createComponent
+import static com.lesfurets.jenkins.unit.declarative.GenericPipelineDeclaration.executeWith
 import static groovy.lang.Closure.*
 
-class StageDeclaration extends GenericPipelineDeclaration {
+class StageDeclaration {
```

Comment:
stage declaration is actually need GenericPipleineDecalration IMHO

Reply:
Does StageDeclaration have all the things a PipelineDeclaration has? I was in doubt about this one...

Reply:
I went back to extending GenericPipelineDeclaration